terraform/modules/tf_stream_alert_monitoring/variables.tf

```diff
@@ -19,3 +19,69 @@ variable "lambda_functions" {
   type    = "list"
   default = []
 }
+
+// Lambda Invocation Error Alarm Settings
+
+variable "lambda_invocation_error_threshold" {
+  default = "0"
```

Comment:
These can (and should, IMO) all be numbers

Reply:
Just being consistent with their examples: https://www.terraform.io/docs/providers/aws/r/cloudwatch_metric_alarm.html